### ISSUE TITLE
Sikkerhedsfeature: auto log-off for instruktører

### ISSUE THREAD

**Issue by plaursen** [janhebnes/startlist.club#68]
For at mindske konsekvensen af en tablet som 'ligger og flyder' med indlogget instruktør, kan brugeren logges af XX minutter efter sidste betjening.

**Comment by janhebnes** [comment]
Jeg vil foreslå det er noget som bliver konfigurerebart, som udgangspunkt
lægger folk også deres log bøger rundt omkring uden at andre kigger løst i
dem.

Det kan styres med tablet låse tiden eller og en fuld dag i bagsædet bliver
lang hvis man efter hver flyvning skal logge på igen...

Det er det renere hvis f.eks tablet skærmen låser adgangen, så kan man
f.eks. finger aftryk og andet hurtigt.

Med hensyn til at man lægger navn til med sit login er en dagsrapport over
ens instruktør aktiviter efter fyraften noget som giver værdi på både audit
og evt. Tastefejl under dagen.

fre. 5. mar. 2021 17.58 skrev plaursen <notifications@github.com>:

**Comment by plaursen** [comment]
Ja, der er ikke så let at klave noget der ikke opleves irriterende.
Problemet er lidt større end med logbøger, da man jo kan redigere.
P.t. har vi ikke noget audit trail på skoleflyvningsdata (#67)

Måske en reminder-popup ved logon som instruktør ? ("Husk at du er logget ind med udvidede rettigheder, lad ikke denne enhed ligge frit tilgængeligt" eller lignende)

**Comment by plaursen** [comment]
Jeg  hørte et forslag om at kunne angive i hvilken rolle man ønsker at logge ind, hvis man er instruktør.

v Login:
Du er instruktær. Vi du
a) logge ind som instruktør. Du skal huske at logge ud igen hvis du lægger enheden fra dig, da du har adgang til data, som kun instruktører må have adgang til
b) Log ind som almindelig pilot. Du har så ikke adgang til skolingsdata.

 - og så slæber man valget med rundt i en cookie.

**Comment by janhebnes** [comment]
Der er lige nu lidt det samme problem for klub administratorer, og der er
det lavet så man kan logge på med sms login, så bliver pilot rettigheden
altid den laveste.

Samme system kunne bruges til v1 på instruktørerne.



lør. 20. mar. 2021 10.52 skrev plaursen ***@***.***>:
